man/mkosi.1

```diff
@@ -276,7 +276,7 @@ or the command line arguments passes \[lq]console=ttyS0 rw\[rq] to the
 kernel in any case.
 Just adding \[lq]console=ttyS0 rw\[rq] would append these two arguments
 to the kernel command line created by lower priority configuration files
-or previous \[en]kernel-command-line command line arguments.
```

Comment:
That file is autogenerated from `mkosi.md`. The fix is rather to put the bare switches in the text in backquotes.

Reply:
And I was wondering why anyone would bother writing  a page in raw roff. I'll replace that patch with a patch for the markdown file and regenerate the man page.